docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:xenial
+
+RUN apt-get update
+RUN apt-get install -y curl
+RUN apt-get install -y lsb-release
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
8 space indenting on curl, lsb-release
4 space on rm -rf

Github formatting won't let me suggest it in any easy way...